var/spack/repos/builtin/packages/opencoarrays/package.py

```diff
@@ -34,20 +34,18 @@ class Opencoarrays(CMakePackage):
     """
 
     homepage = "http://www.opencoarrays.org/"
-    url      = "https://github.com/sourceryinstitute/opencoarrays/releases/download/1.6.2/OpenCoarrays-1.6.2.tar.gz"
+    url      = "https://github.com/sourceryinstitute/opencoarrays/releases/download/1.7.4/OpenCoarrays-1.7.4.tar.gz"
 
+    version('1.7.4', '85ba87def461e3ff5a164de2e6482930')
     version('1.6.2', '5a4da993794f3e04ea7855a6678981ba')
 
     depends_on('cmake', type='build')
     depends_on('mpi')
 
     provides('coarrays')
 
-    def install(self, spec, prefix):
-        with working_dir('spack-build', create=True):
-            args = std_cmake_args
-            args.append("-DCMAKE_C_COMPILER=%s" % self.spec['mpi'].mpicc)
-            args.append("-DCMAKE_Fortran_COMPILER=%s" % self.spec['mpi'].mpifc)
-            cmake('..', *args)
-            make()
-            make("install")
+    def cmake_args(self):
+        args = []
```

Comment:
Sorry, I don't follow your comment. 

I'm looking at `lib/spack/spack/build_systems/cmake.py` - [`cmake_args`](https://github.com/LLNL/spack/blob/develop/lib/spack/spack/build_systems/cmake.py#L94) is the method to me overridden and its output gets appended to [`std_cmake_args`](https://github.com/LLNL/spack/blob/develop/lib/spack/spack/build_systems/cmake.py#L63) on [`cmake.py:104`](https://github.com/LLNL/spack/blob/develop/lib/spack/spack/build_systems/cmake.py#L104). 

Sorry again if I totally misunderstood your point!